Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2716,7 +2761,14 @@ private protected virtual void OnDisable()
         SceneView.duringSceneGui -= OnSceneViewGUI;
 #endif
     }
-#endregion
+
+    private void Update() {
+#if AT_USE_HDRP
+        UpdatePathTracingState();
+#endif
```

Comment:
Does the number of servers in the scene affect performance?

Reply:
No, in theory, because `Object.FindObjectsOfType` is only done once per frame at most